packages/osd-babel-preset/node_preset.js

```diff
@@ -40,7 +40,7 @@ module.exports = (_, options = {}) => {
             // `nvm install 8 && node ./src/cli` will run OpenSearch Dashboards
             // in node version 8 and babel will stop transpiling async/await
             // because they are supported in the "current" version of node
-            node: 'current',
+            node: 14,
```

Comment:
Why do we need this change?

Reply:
Left as `current`, we could be building with Node 18 and producing artifacts that are not compatible with Node 14, while claiming to allow users to use Node 14 with OSD. This makes sure any built and released artifact will work on Node 14.

Reply:
I figured as much, what i worry is that this means that we dont benefit from any of the node 18 updates just because we build on node 14. I think we should build by default on the latest version of node and anyone who wants to use an older version can but they need to rebuild the artifacts themselves. We shouldnt build for 14 by default.

Reply:
Since we package the artifacts of a release, consumers of those artifacts don't have the option of rebuilding. Also, since this the CommonJS part of the code that runs on the server, payload size it not a worry.